docs/user-guide-for-quickstart.md

```diff
@@ -0,0 +1,133 @@
+---
+id: user-guide-for-quickstart
+title: Quickstart Tour
+sidebar_label: Tour
+---
+
+StreamPipes is a framework that enables domain experts to model and execute stream processing pipelines in a big data infrastructure.
+The graphical user interface of StreamPipes is a web application that provides an easy to use solution for domain experts.
+
+Quickstart Mode, ideal for first-time users, allows users to interact with pre-set pipelines, dashboards, and data views to experience StreamPipes' functionalities for IIoT. 
+This page provides an overview of StreamPipes' features in Quickstart Mode, guiding users on how to utilize them effectively.
+
+
+First of all, a user needs to log in to the system. The default login credentials are ``admin@streampipes.apache.org``, password ``admin``
+The credentials for the user are specified during the installation process.
+
+
+## Home
+After logging in, the user is redirected on to the home page.
+The home page gives an overview over the different features available in the StreamPipes UI.
+
+On the left, the navigation menu can be seen.
+You can either use the icons on the left side or click on the menu icon on the
+top left to open the details view of the navigation menu.
+
+<div class="my-carousel docs-carousel">
+    <img src="/img/quickstart/user-guide/01_home.png" alt="Home"/>
+</div>
+
+
+## Pipelines
```

Comment:
Can we please start with the Connect module? Simply switch the ordering

Reply:
Hi, @bossenti ! I have put the `Connect` module as the first module after `Home`.